tools/gifenc.sh

```diff
@@ -6,7 +6,8 @@
 palette="/tmp/palette.png"
 
 #filters="fps=15,scale=320:-1:flags=lanczos"
-filters="fps=25,scale=640:-1:flags=lanczos"
+#filters="fps=25,scale=640:-1:flags=lanczos"
+filters="fps=25,scale=1012:-1:flags=lanczos"
```

Comment:
Unrelated change, or at least undocumented in commit text.

Reply:
For ov5647 and imx219 the modes match camera documentation.
For imx477 they did not.
Do you want to take changed mode numbers (0123->02103) as I did to match Raspberry camera documentation?
If so, how can I modify my changelist into 2 (mode renumbering / 4 HQ tools+gifenc.sh change) without deleting my fork, forking new and do changes based on your latest from fresh?
I just commited camera_i2c fix on my fork before reading your comment, and the commit contains camera_i2c i2c device change as well as changes for the 4 tools. In case you want the camera_i2c change, I will need to split that as well.